test/framework/src/main/java/org/opensearch/index/engine/EngineTestCase.java

```diff
@@ -111,8 +111,7 @@
 import org.opensearch.index.seqno.SequenceNumbers;
 import org.opensearch.index.shard.ShardId;
 import org.opensearch.index.store.Store;
-import org.opensearch.index.translog.Translog;
-import org.opensearch.index.translog.TranslogConfig;
+import org.opensearch.index.translog.*;
```

Comment:
Avoid `*` in imports (ideally, spotless check should report this, created an issue for the same: https://github.com/opensearch-project/OpenSearch/issues/3767)